lib/View/courseoverview_screen.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'package:gocast_mobile/View/mycourses_screen.dart';
+import 'package:gocast_mobile/View/publiccourses_screen.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/View/utils/viewall_button_view.dart';
+import 'settings_screen.dart';
+import 'package:gocast_mobile/View/utils/course_card_view.dart';
+
+final currentIndexProvider = StateProvider<int>((ref) => 0);
+
+class CourseOverview extends ConsumerWidget {
+  const CourseOverview({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white, // Replace with the exact color
+        title: const Text(
+          'GoCast',
```

Comment:
Hardcoded String ´GoCast´.

Hardcoded Strings and Assets should be centralized in one file/class --> This will make it easier making it easier to update.

You can create a Dart class where we store all our constants.

class AppStrings {
  static const String appTitle = 'GoCast';
  static const String myCoursesTitle = 'My Courses';
  static const String publicCoursesTitle = 'Public Courses';
 ...... and so on.....
}
class AppImages {
  static const String courseImage1 = 'assets/images/course1.png';
  static const String courseImage2 = 'assets/images/course2.png';
  .....................
}
.....................................

Reply:
I think this we can do in next PR

Reply:
Ok I ll refine the task for it and the others.

Reply:
done